README.md

```diff
@@ -1,2 +1,25 @@
 # pyrsh
 Remote shell in python
+
+to run in cmd you need to enter
+`pyrsh.py type command user password host port`
```

Comment:
@GitSergeyGit password should not be expected as input parameter. Also it should be removed from line 14